src/cli.js

```diff
@@ -408,12 +412,12 @@ function runCommandFromCli(args: options.Args, flags: options.Flags) {
 export default async function cli(argv: Array<string>, exit: boolean = false) {
   let start = Date.now();
 
-  let { pkg, input, flags } = meow({
+  let { pkg, input, flags } = meow('', {
```

Comment:
Whats the string here for @Blasz ? 🤔 💭

Reply:
The `meow` flow typedefs don't support the variant without the helptext as the first arg so I just gave it an empty string.
This is what `meow` itself does when passing the options as the first argument anyway. 
The reason why we don't pass the full help text is we're not relying on meow to display help text.